.github/workflows/wheel_tests_and_release.yml

```diff
@@ -22,6 +22,7 @@ jobs:
     permissions:
       contents: write # for softprops/action-gh-release to create GitHub release
       id-token: write # IMPORTANT: this permission is mandatory for trusted publishing
+      attestations: write # for GitHub attestations
```

Comment:
Oh, so they do require write access? :upside_down_face:

Reply:
Yeah, they require the following permissions:

```yaml
permissions:
  id-token: write
  attestations: write
```

c.f. https://github.com/actions/attest-build-provenance?tab=readme-ov-file#usage